vendors/tpl-systemes/drivers/birdy-slim-iot/examples.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "description": "Example1: uplink containing message",
+    "type": "uplink",
+    "input": {
+      "bytes": "5765206E65656420796F75204153415021",
+      "fPort": 10,
+      "recvTime": "2020-08-02T20:00:00.000+05:00"
+    },
+    "output": {
+      "message": "We need you ASAP!",
```

Comment:
examples should be adapted after fixing the code in index.js following the review comments